python/run/miniAODToNTuple_cfg.py

```diff
@@ -150,6 +152,7 @@
 else:
     process.makingNTuples = cms.Path(
         # process.metFilters *
+        process.smearedElectrons *
```

Comment:
So for real data the electrons are scaled instead of smeared - Both the scaling and smearing are performed by the same module which takes in an isMC bool.